index_review.js

```diff
@@ -0,0 +1,257 @@
+						};
+
+						// Update of PB plugin integration connection
+						axios(pbIntegrationConfig)
+							.then(function (response) {
+								console.log("Feature connected to Gitlab.");
+							})
+							.catch(function (error) {
+								console.log(error.message);
+							});
+					})
+					.catch(function (error) {
+						console.log("Error when creating GitLab issue:", error);
+					});
+			})
+			.catch(function (error) {
+				console.log("Error when getting Productboard feature:", error);
+			});
+		res.status(200).end();
+
+		// If button trigger is cancel or dismiss, set PB plugin connection to initial state (basically disconnected)
+	} else {
+		res.json({
+			data: {
+				connection: {
+					state: "initial",
+				},
+			},
+		});
+		res.status(200).end();
+		console.log("Feature is unlinked");
+	}
+});
+
+// Endpoint for requests from Gitlab for status updates in plugin integration column
+app.post("/gitlab-webhook", async (req, res) => {
+	// Grab information about the GitLab issue
+	const gitlabIssueId = req.body.object_attributes.id;
+	const gitlabIssueStatus = req.body.object_attributes.state;
+	const gitlabIssueURL = req.body.object_attributes.url;
+
+	console.log(
+		"The issue ID is:",
+		gitlabIssueId,
+		"and the state is:",
+		gitlabIssueStatus
+	);
+	const pbTooltip = "Issue " + gitlabIssueId;
+	let pbTooltipColor = "";
+
+	// Update plugin integration column color depending on Gitlab status
+	if (gitlabIssueStatus === "opened") {
+		pbTooltipColor = "blue";
+	} else {
+		pbTooltipColor = "green";
+	}
+
+	// Setup update of plugin integration connection
+	var pbIntegrationConfig = {
+		method: "get",
+		url: `https://api.productboard.com/plugin-integrations/${pbIntegrationID}/connections/`,
+		headers: {
+			"X-Version": "1",
+			Authorization: productboardToken,
+			"Content-Type": "application/json",
+		},
+	};
+
+	const pbBaseURL = "https://api.productboard.com";
+	let offset = 0;
+	let featureResult = [];
+
+	// Loop through features in PB to find the right feature based Gitlab issue ID
+	axios(pbIntegrationConfig)
+		.then(function (response) {
+			// Find the right feature to update
+			featureResult = response.data.data.filter((obj) => {
+				return obj.connection.tooltip === pbTooltip;
+			});
+			console.log(
+				"Feature result is",
+				featureResult,
+				"and its length is",
+				featureResult.length
+			);
+
+			// Configre the body of the update of the plugin integration column
+			const pbUpdateFromGitlabData = JSON.stringify({
+				data: {
+					connection: {
+						state: "connected",
+						label: gitlabIssueStatus,
+						hoverLabel: `Issue ${gitlabIssueId}`,
+						tooltip: `Issue ${gitlabIssueId}`,
+						color: pbTooltipColor,
+						targetUrl: gitlabIssueURL,
+					},
+				},
+			});
+
+			// Once feature is found, setup the request to update the plugin integration connection in PB
+			const pbFeatureID = featureResult[0].featureId;
+			var pbPluginUpdate = {
```

Comment:
Don't use `var`, use `const` whenever possible and if you need it to be mutable then go with `let` - looks like you already know this by looking at the rest of the file, so just a reminder :)